grails-app/controllers/au/org/ala/specieslist/PublicController.groovy

```diff
@@ -22,30 +22,20 @@ class PublicController {
     }
 
     def speciesLists(){
-        if (params.isSDS) {
-            // work around for SDS sub-list
-            redirect(action:'sdsLists')
-            return
-        }
-        params.max = Math.min(params.max ? params.int('max') : 25, 100)
+        params.max = Math.min(params.max ? params.int('max') : 25, 1000)
```

Comment:
I see that the check for SDS lists as well as _not_ showing private lists has been moved into the service, which is a good idea. I'm wondering if a few integration/functional tests should be added that test those conditions, to ensure they are handled correctly and don't break when code is changed?

Reply:
I'm not 100% sure the `isSDS` case is handled in the service... hard to tell just reading code but I'm assuming its now a facet that is just like the other fields...

Reply:
ive put back the isSDS=true check, but removed the redirect to public/sdsLists - that URL now redirects to public/speciesLists?isSDS=eq:true. Hopefully this is more consistent for users - one UI with filters on the left hand side.